themes/login-forms/templates/SilverStripe/Forms/CheckboxField.ss

```diff
@@ -1 +1 @@
-<input $getAttributesHTML('title', 'data-toggle') />
+<input $getAttributesHTML('title') />
```

Comment:
No need to include `data-toggle` here.

```suggestion
<input $getAttributesHTML('title', 'data-toggle') />
```

Reply:
:-) I think I removed it and your recommendation is to add it back in?

Reply:
Including it as an argument will exclude it from being rendered. Check the function definition.

Reply:
yeah but we're no longer adding data-toggle - was removed from the framework pr remember